integration_tests/e2e/induction/createCheckYourAnswersChangeLinks.cy.ts

```diff
@@ -280,6 +280,65 @@ context(`Change links on the Check Your Answers page when creating an Induction`
       .chooseAdditionalTraining(AdditionalTrainingValue.CSCS_CARD)
       .submitPage()
 
+    // Change a previous work experience that has already been added to the Induction
```

Comment:
Add the new page interactions to the cypress test that tests "creating a Short question set induction; get all the way through to Check Your Answers; then test the Change links for each question work"